### PR TITLE
Updates to upstream to sync with downstream content

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
/kind documentation
Signed-off-by: Srivaralakshmi srr@redhat.com

# Changes
Updates to upstream to sync with downstream content

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] [Docs](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#docs) 
  included if any changes are user facing
- [ ] [Tests](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#tests)
  included if any functionality added or changed. For bugfixes please include tests that can catch regressions
- [ ] All acceptance test scenarios included in the PR which verifies a bugfix or a requested feature reported by a non-member are tagged with `@external-feedback` tag.
- [x] Follows the [commit message standard](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#commits)

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@pmacik, @dperaza4dustbit, @baijum, and @sadlerap: Requesting review. I have streamlined the content, and improved the flow and placement so that the upstream content is in sync with the downstream content.

File: docs/userguide/modules/exposing-binding-data/pages/intro-expose-binding.adoc [hunk 1]
```diff
@@ -1,84 +1,121 @@
...
-
-* Expose a string from a resource
-* Expose a constant value
-* Expose an entire ConfigMap or Secret that is referenced from a
+Application developers need access to backing services to build and connect workloads. Connecting workloads to backing services is always a challenge because each service provider requires a different way to access their secrets and consume them in a workload.
+
+The {servicebinding-title} enables application developers to easily bind workloads together with operator-managed backing services, without any manual procedures to configure the binding connection. For the {servicebinding-title} to provide the binding data, as an Operator provider or user who creates backing services, you must expose the binding data to be automatically detected by the {servicebinding-title}. Then, the {servicebinding-title} automatically collects the binding data from the backing service and shares it with a workload to provide a consistent and predictable experience.
+
+[#methods-of-exposing-binding-data]
+= Methods of exposing binding data
```
Comment (pmacik) [inline docs/userguide/modules/exposing-binding-data/pages/intro-expose-binding.adoc:9]:
```suggestion
== Methods of exposing binding data
```

This should be a level 2 header

File: config/rbac/percona_mongodb_clusterrole.yaml [hunk 2]
```diff
@@ -12,4 +12,5 @@ rules:
       - perconaservermongodbs/status
     verbs:
       - get
+      - watch
```
Comment (pmacik) [inline config/rbac/percona_mongodb_clusterrole.yaml:15]: Why do you change the clusterrole here, is that on purpose?

If it is not intentional or required it should be removed from the PR.

Comment (Srivaralakshmi) [reply config/rbac/percona_mongodb_clusterrole.yaml:15]: I just copied the yaml from the downstream.. that one has captured the watch verb in the list as per @baijum's suggestion, but the upstream doc was missing this and I had to update it. So yes, it is intentional.

File: docs/userguide/modules/binding-workloads-using-sbo/pages/unbinding-workloads.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+OR
++
+[source,terminal]
+----
+$ oc delete ServiceBinding <.metadata.name>
+----
++
+where `<.metadata.name>` is the name of the `ServiceBinding` resource.
++
+.Examples
```
Comment (pmacik) [inline docs/userguide/modules/binding-workloads-using-sbo/pages/unbinding-workloads.adoc:22]:
```suggestion
where `<.metadata.name>` is the name of the `ServiceBinding` resource.
+
[NOTE]
====
`ServiceBinding` custom resource (CR) belongs to the `binding.operators.coreos.com/v1alpha1` API. To unbind a workload from a backing service bound using a `ServiceBinding` custom resource (CR) from the `servicebinding.io/v1alpha3` API:

[source,terminal]
----
$ kubectl delete servicebindings.servicebinding.io <.metadata.name>
----

or

[source,terminal]
----
$ oc delete servicebindings.servicebinding.io <.metadata.name>
----
====
+
.Examples
```

Similarly as we mention both APIs in the `Binding workloads using Service Binding Operator` section, we should mention both APIs here, especially when the command is different for each.

**Comment by pmacik** [comment]
@Srivaralakshmi I'me done with my reviews

File: docs/userguide/modules/exposing-binding-data/pages/intro-expose-binding.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply docs/userguide/modules/exposing-binding-data/pages/intro-expose-binding.adoc:9]: Fixed. Good catch! Thank you :)

File: docs/userguide/modules/binding-workloads-using-sbo/pages/unbinding-workloads.adoc (hunk 3 above)
Comment (Srivaralakshmi) [reply docs/userguide/modules/binding-workloads-using-sbo/pages/unbinding-workloads.adoc:22]: Fixed

Comment (Srivaralakshmi) [reply docs/userguide/modules/binding-workloads-using-sbo/pages/unbinding-workloads.adoc:22]: Fixed

**Comment by Srivaralakshmi** [comment]
@pmacik: Thanks for the suggestions. Very helpful. I have incorporated all of them and checked the rendering, all good now. PTAL and provide your feedback. Thanks!

**Pull request merged**